ruby-training/trainees/alan/test_katas/01/math_maddness/math_maddness.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec/autorun'
```

Comment:
You aren't using any methods or classes from this gem in this file, this line can (should) be removed.